src/AsyncGenerator.yml

```diff
@@ -183,7 +183,7 @@
     scanForMissingAsyncMembers:
     - all: true
 - filePath: NHibernate.Test/NHibernate.Test.csproj
-  targetFramework: netcoreapp2.0
+  targetFramework: netcoreapp2.1
```

Comment:
I have upgrade only the test project. Unfortunately, it seems the async generator does not support generating a project for core 2.1 which depends on core 2.0 libs : it tries to compile the 2.0 libs as 2.1 libs.
@maca88 

A workaround would be to add a core 2.1 target to the NHibernate libs too, but just for being able able to runs tests in VS2022, it sounds overkill to me.

Reply:
> Unfortunately, it seems the async generator does not support generating a project for core 2.1 which depends on core 2.0 libs : it tries to compile the 2.0 libs as 2.1 libs.

I do apologize as I didn't saw this earlier. Unfortunately, I didn't found a workaround yet that would not require to add `netcoreapp2.1` to other projects.

> A workaround would be to add a core 2.1 target to the NHibernate libs too

Another workaround would be to use `netcoreapp3.1` for the test project which works fine as opposed to `netcoreapp2.1`.

Reply:
No apologies are required! We are all contributors on our free time, no one is supposed to have any "SLA" with short response time, ...

Thanks for your investigation. So you say it work well to generate async code for a project targeting 3.1 and referencing project targeting 2.0?

That would be better than my current workaround, targeting both 2.0 and 2.1 in the test project and generating async for 2.0. @hazzik, are you agree?

Reply:
> So you say it work well to generate async code for a project targeting 3.1 and referencing project targeting 2.0?

That is correct, [here](https://github.com/maca88/nhibernate-core/commit/a86287916d2e35045428f8bde1ded3e8a1f63feb) you can find the changes that I used to test `3.1`.